meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -391,34 +380,13 @@ public int hashCode() {
     }
   }
 
-  private static MetaClientException translateZkExceptionToMetaclientException(ZkException e) {
-    if (e instanceof ZkNodeExistsException) {
-      return new MetaClientNoNodeException(e);
-    } else if (e instanceof ZkBadVersionException) {
-      return new MetaClientBadVersionException(e);
-    } else if (e instanceof ZkTimeoutException) {
-      return new MetaClientTimeoutException(e);
-    } else if (e instanceof ZkInterruptedException) {
-      return new MetaClientInterruptException(e);
-    } else {
-      return new MetaClientException(e);
-    }
-  }
-
-  private static EntryMode convertZkEntryMode(long ephemeralOwner) {
-    EphemeralType zkEphemeralType = EphemeralType.get(ephemeralOwner);
-    switch (zkEphemeralType) {
-      case VOID:
-        return EntryMode.PERSISTENT;
-      case CONTAINER:
-        return EntryMode.CONTAINER;
-      case NORMAL:
-        return EntryMode.EPHEMERAL;
-      // TODO: TTL is not supported now.
-      //case TTL:
-      //  return EntryMode.TTL;
-      default:
-        throw new IllegalArgumentException(zkEphemeralType + " is not supported.");
-    }
+  @Override
+  public List<OpResult> transactionOP(Iterable<Op> iterable) {
```

Comment:
neat: iterable looks bad variable name. Do you want to change it something like iterableOps or just ops or anything which tells that these are ops list.

Reply:
Good catch. The metaclient interface has it as ops as well so it only makes sense to be consistent with the naming too.